src/EnergyPlus/HVACStandAloneERV.cc

```diff
@@ -2047,4 +2048,96 @@ namespace EnergyPlus::HVACStandAloneERV {
         return GetStandAloneERVReturnAirNode;
     }
 
+    bool StandAloneERVAFNExceptions(EnergyPlusData &state, int const NodeNumber)
+    {
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Jeremy Lerond & Yan Chen
+        //       DATE WRITTEN   January 2021
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
```

Comment:
Just a quick initial comment. We don't add author comments to new code anymore. Please delete these lines and similar ones in WaterThermalTanks. And this will require a unit test. I've marked this as a draft PR until that is completed. Then you can flip back to ready for review (that button shows at the bottom of the PR).

Reply:
Thanks @mjwitte for the review! Good to know, I just removed these lines, I was using `GetZoneDehumidifierNodeNumber` as a template for formatting (which includes them). We've been putting the tests on hold as Jason mentioned by email that the PR has issues. Should we go ahead and add them anyway? Or, should we keep holding off.